chcemvediet/apps/inforequests/admin.py

```diff
@@ -3,11 +3,25 @@
 from django.contrib import admin
 
 from poleno.utils.misc import decorate
-from poleno.utils.admin import simple_list_filter_factory, admin_obj_format
+from poleno.utils.admin import (simple_list_filter_factory, admin_obj_format,
+                                ReadOnlyAdminInlineMixin)
 
 from .models import Inforequest, InforequestDraft, InforequestEmail, Branch, Action
 
 
+class BranchInline(ReadOnlyAdminInlineMixin, admin.TabularInline):
+    model = Branch
+    fields = [
+            decorate(
+                lambda o: admin_obj_format(o, u'{obj.pk}'),
```

Comment:
Mozno by bolo citatelnejsie, ak by namiesto PK sa zobrazovalo `<Branch: PK>`. To sa vypise defaultne, ked parameter `u'{obj.pk}'` vynechas. Vid zdrojaky `admin_obj_format`. Analogicky pri `ActionInline`.

Reply:
:ok: